packages/commons-server/src/libs/response-rules-interpreter.ts

```diff
@@ -170,6 +170,10 @@ export class ResponseRulesInterpreter {
 
     const parsedRuleValue = this.parseValue(rule.value);
 
+    if (rule.operator === 'array_includes' && rule.modifier) {
+      return Array.isArray(value) && value.includes(parsedRuleValue);
```

Comment:
This currently works only with arrays of strings as the rule's value field is always containing a string. 
Could you make it work with numbers, booleans and null too?

If could be something like this (not tested): 
`return Array.isArray(value) && value.some(element => String(element) === parsedRuleValue);`

Reply:
Noted